looker_blueprints/helpers.py

```diff
@@ -0,0 +1,45 @@
+import os
+import shipyard_utils as shipyard
+import looker_sdk
+from looker_sdk import api_settings
+
+# create Artifacts folder paths
+base_folder_name = shipyard.logs.determine_base_artifact_folder(
+    'looker')
+artifact_subfolder_paths = shipyard.logs.determine_artifact_subfolders(
+    base_folder_name)
+shipyard.logs.create_artifacts_folders(artifact_subfolder_paths)
+
+
+class MyApiSettings(api_settings.ApiSettings):
+    def __init__(self, *args, **kw_args):
+        self.base_url = kw_args.pop('base_url')
```

Comment:
Not sure where to throw this comment, but I'd like to make sure we handle user error and add in `https://` if it's not included.

Reply:
This is done